Content.Server/GameObjects/EntitySystems/GravitySystem.cs

```diff
@@ -19,42 +16,34 @@
 namespace Content.Server.Interfaces.GameObjects.Components.Interaction
 {
     [UsedImplicitly]
-    public class GravitySystem: EntitySystem
+    internal sealed class GravitySystem: EntitySystem
```

Comment:
super nitpicky: please move the colon.
`GravitySystem : EntitySystem`

Reply:
```suggestion
    internal sealed class GravitySystem : EntitySystem
```